djangocms_snippet/templatetags/snippet_tags.py

```diff
@@ -2,7 +2,7 @@
 """
 Snippet template tags
 """
-import six
+from django.utils import six
```

Comment:
Please move this line under ``from django import template``

Reply:
That's done!